src/components/portfolio/DashboardCards.tsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Card,
+  CardActions,
+  CardContent,
+  GridList,
+  GridListTile,
+  IconButton,
+  makeStyles,
+  Tooltip,
+  Typography,
+} from '@material-ui/core';
+import AddBoxIcon from '@material-ui/icons/AddBox';
+// TODO: possible rename of shared props types
+import {
+  DashboardTableProps,
+  DashboardTableRowProps,
+  DashboardTableActions,
+} from './DashboardTable';
```

Comment:
I don't think it's a good thing to import from `DashboardTable`. `DashboardCards` should be an alternative to the table and not depend on it. Actually, `DashboardTableRow` and its props should not be exported. It's an implementation detail of `DashboardTable` (that's my bad).

If the actions are the same for the cards and table component, it would make sense to create a new component that is used by both table and cards.

Reply:
I put the actions in a separate file.